config/install/field.storage.node.field_residents.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.field_residents
+field_name: field_residents
+entity_type: node
+type: entity_reference
+settings:
+  target_type: node
+module: core
+locked: false
+cardinality: 1
```

Comment:
This field should accept "Unlimited" values.
<img width="515" alt="planet_field_residents" src="https://github.com/notS0Solo/swapimigration/assets/1434979/da430d86-d434-4f84-86b4-3eb022d9a7c6">

Reply:
Fixed